Changes in module 'nf-core/sambamba/markdup'
--- modules/nf-core/sambamba/markdup/main.nf
+++ modules/nf-core/sambamba/markdup/main.nf
@@ -12,6 +12,7 @@
 
     output:
     tuple val(meta), path("*.bam"), emit: bam
+    tuple val(meta), path("*.txt"), emit: txt
     path "versions.yml"           , emit: versions
 
     when:
@@ -27,7 +28,8 @@
         $args \\
         -t $task.cpus \\
         $bam \\
-        ${prefix}.bam
+        ${prefix}.bam \\
+        2> ${prefix}.txt
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":

************************************************************
